### PR TITLE
added CSGO Inventory private check

### PR DESCRIPTION
**Pull request by LemonPAKA**
tested in csgo and it works, but didn't test in TF2,DOTA2 and portal2..I think it should work, because I didn't change any code about this part

### CODE REVIEW

**Review by rumblefrog — CHANGES_REQUESTED** [review]
I can't review this PR as all the lines are overwritten, and I can't see what was changed. Can you fix this?

**Comment by LemonPAKA** [comment]
I think I fixed the overwritten problem...

**Comment by rumblefrog** [comment]
You should be force pushing the commit to the branch, the indentation changes are unnecessary and makes it harder for me to review.

File: Anti_Private.sp [hunk 1]
```diff
@@ -278,55 +279,92 @@ void ParseProfile(const char[] sBody, int iClient)
...
 			default:
 				return;
 		}
-		
-		char SteamID[64];
-	
-		GetClientAuthId(iClient, AuthId_SteamID64, SteamID, sizeof SteamID);
-	
-		if (STEAMWORKS_AVAILABLE())
+		if(GetEngineVersion()==Engine_CSGO)
```
Comment (rumblefrog) [inline Anti_Private.sp]: Is there a reason why you are duplicating SteamWorks and SteamTools check just for CSGO.

File: Anti_Private.sp [hunk 2]
```diff
@@ -278,55 +279,92 @@ void ParseProfile(const char[] sBody, int iClient)
...
-		
+			
+		char SteamID[64];
+	
+		GetClientAuthId(iClient, AuthId_SteamID64, SteamID, sizeof SteamID);
+
 		switch (GetEngineVersion())
 		{
 			case Engine_TF2, Engine_DOTA, Engine_Portal2: {}
+			case Engine_CSGO:
```
Comment (rumblefrog) [inline Anti_Private.sp:296]: The InventoryURL should be defined on start and engine be checked here

**Review by rumblefrog — CHANGES_REQUESTED** [review]

**Comment by LemonPAKA** [comment]
I don't know how to defined InventoryURL as other games...so I just made a sample for it...and already pushed the commit to the branch

**Comment by rumblefrog** [comment]
Yeah, I'm not sure why you pushing additional commits to restore and redo. I wanted you to do a **force push** to your branch.

**Pull request closed without merging**